tests/suites/SocketServerTest.php

```diff
@@ -168,4 +168,25 @@ public function testContext(): void
         $this->assertEquals('c', $server->getContext()->getOption('a', 'b'));
         $server->close();
     }
+
+    public function testAcceptWithInvalidSslContextShowsPreviousException(): void
+    {
+        $uri = new Uri('tls://0.0.0.0:8010');
+        $context = new Context();
+        $context->setOptions([
+            'ssl' => [
+                'local_cert' => '/nonexistent/cert.pem',
+                'local_pk' => '/nonexistent/key.pem',
+            ],
+        ]);
+
+        $server = new SocketServer($uri, $context);
+        $server->setBlocking(true);
+
+        $this->expectException(StreamException::class);
+        $this->expectExceptionCode(StreamException::SERVER_ACCEPT_ERR);
+        $this->expectExceptionMessage("Could not accept on socket. (stream_socket_accept(): Unable to set private key file `/nonexistent/key.pem')");
```

Comment:
Using PSR code standard, line must be max 120 characters long.
```suggestion
        $this->expectExceptionMessage(
            "Could not accept on socket. "
            . "(stream_socket_accept(): Unable to set private key file `/nonexistent/key.pem')"
        );
```